CHANGELOG.md

```diff
@@ -6,40 +6,28 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 3.1.2
+## 4.0.2
 
--   Fixes bug that caused values from previously created vault item to be
-    pre-filled when creating the next item.
--   Fixes attachment previews on Android
+-   (feature): New Italian translation (grazie @coluzziandrea)
+-   (feature): Updated multiple dependencies for security and reliability
+    improvements
+-   (fix): Fullscreen notes on iOS are now usable
+-   (fix): Allow changing master password
```

Comment:
For fixes, I prefer describing the original problem, rather than the effect of the fix. Something along the lines of:

- Fixed various usability issues with full-screen note editing on iOS
- Fixed problem with changing the master password